nixos/modules/services/backup/borgmatic.nix

```diff
@@ -5,54 +5,71 @@ with lib;
 let
   cfg = config.services.borgmatic;
   settingsFormat = pkgs.formats.yaml { };
+
+  cfgType = with types; submodule {
+    freeformType = settingsFormat.type;
+    options.location = {
+      source_directories = mkOption {
+        type = listOf str;
+        description = mdDoc ''
+          List of source directories to backup (required). Globs and
+          tildes are expanded.
+        '';
+        example = [ "/home" "/etc" "/var/log/syslog*" ];
+      };
+      repositories = mkOption {
+        type = listOf str;
+        description = mdDoc ''
+          Paths to local or remote repositories (required). Tildes are
+          expanded. Multiple repositories are backed up to in
+          sequence. Borg placeholders can be used. See the output of
+          "borg help placeholders" for details. See ssh_command for
+          SSH options like identity file or port. If systemd service
+          is used, then add local repository paths in the systemd
+          service file to the ReadWritePaths list.
+        '';
+        example = [
+          "user@backupserver:sourcehostname.borg"
```

Comment:
```suggestion
          "user@backupserver:example.org"
```

Reply:
That's not an example domain, but a folder name instead.

Maybe we can update this list to an up-to-date version https://github.com/borgmatic-collective/borgmatic/commit/3be99de5b1eb189e0166d5dec8fbe5058fe85bf5

```
ssh://user@backupserver/./sourcehostname.borg
ssh://user@backupserver/./{fqdn}
/var/local/backups/local.borg
```